### ISSUE TITLE
[2.0.0-beta.2] nitro remove doesn't update `~/.nitro/nitro.yaml`

### ISSUE THREAD

**Issue by tomdavies** [craftcms/nitro#238]
### Description

Running `nitro remove` removes the containers & db for a site, but doesn't appear to remove the entry in the `sites` key from `~/.nitro/nitro.yaml`

### Steps to reproduce

1. `nitro create nitro-test`
2. `nitro remove nitro-test`
3. `cat ~/.nitro/nitro.yaml`

```yaml
databases:
    - engine: mysql
      version: "8.0"
      port: "3306"
services:
    dynamodb: false
    mailhog: false
    minio: false
    redis: true
sites:
    - hostname: nitro-test.nitro
      path: ~/code/kind/nitro-test
      version: "7.4"
      webroot: web
      xdebug: false
```

### Additional info

- Nitro version: 2.0.0-beta.2
- Docker version: 3.1.0
- MacOS Mojave 10.14.6
- Installed nitro via homebrew

**Comment by jasonmccallister** [comment]
Can reproduce, will take a look. However, as a work around, you can run `nitro edit` to remove the site and `nitro apply` will handle the rest.

**Comment by jasonmccallister** [comment]
We just released 2.0.0-beta.6 with this change, should be available shortly!
